src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -50,6 +50,15 @@ std::ostream &operator<<(std::ostream &lhs, const test_extent_record_t &rhs) {
 	     << ", refcount=" << rhs.refcount << ")";
 }
 
+template<>
```

Comment:
This specialization lacks `#if FMT_VERSION >= 90000`. Is this intentional?

Reply:
i wanted to drop the operator<<. but forgot to remove it. let me test if we actually need it.

Reply:
tested. it compiles. so operator<< is now dropped.